internal/pkg/manifest/validate.go

```diff
@@ -1740,6 +1745,9 @@ func isValidSubSvcName(name string) bool {
 }
 
 func validateWindows(opts validateWindowsOpts) error {
+	if aws.BoolValue(opts.readOnlyFS) {
+		return errors.New(`'ReadOnlyFS' can not be set to true when deploying a Windows container`)
```

Comment:
nit: technically, our field name in the manifest is `readonly_fs` :) The [%q fmt verb](https://pkg.go.dev/fmt#hdr-Printing) adds double quotes automatically for the string.
```suggestion
		return fmt.Errorf(`%q can not be set to 'true' when deploying a Windows container`, "readonly_fs")
```